optaplanner-examples/src/test/java/org/optaplanner/examples/cheaptime/score/CheapTimeConstraintProviderTest.java

```diff
@@ -46,11 +48,8 @@ class CheapTimeConstraintProviderTest {
     private static final Task TASK_2 = new Task(2, PERIOD_1, PERIOD_3, 1, 3_000_000,
             REQUIREMENT_TASK2_RESOURCE0, REQUIREMENT_TASK2_RESOURCE1, REQUIREMENT_TASK2_RESOURCE2);
 
-    private final ConstraintVerifier<CheapTimeConstraintProvider, CheapTimeSolution> constraintVerifier =
-            ConstraintVerifier.build(new CheapTimeConstraintProvider(), CheapTimeSolution.class, TaskAssignment.class);
-
-    @Test
-    void startTimeLimitsFrom() {
+    @ConstraintProviderTest
```

Comment:
This is a lot of JUnit 5 fanciness. See `@ConstraintProviderTest` for explanation.